docs/changelog_3_1_0.rst

```diff
@@ -1,8 +1,41 @@
 .. v3.1.0 Changelog
 
-================
+####################
+v3.1.0 Release Notes
+####################
+
+----------------------
+Mongo Driver Migration
+----------------------
+
+Due to the required changes of the Mongo driver for Config, all existing Mongo users will need to
+complete the below instructions to continue to use Mongo after updating to 3.1.
+This includes **all** users, regardless of any prior migration attempt to a development version of
+3.1.
+
+ #. Upgrade to 3.1 (duh)
```

Comment:
```suggestion
 #. Upgrade to 3.1
```
I'd rather leave off the `(duh)` in case we ever have an issue where we need something done prior to an upgrade (though I hope that doesn't come up)